### ISSUE TITLE
Create a staging PR and production PR

### ISSUE THREAD

**Issue by Spoutnik97** [googleapis/release-please#2370]
Hi,
I would like to setup a workflow to deploy my applications (monorepo) in staging and production with release-please.

What are the best practices?
I thought about : 
1. create a release-please-config.staging with:
```
{
  "prerelease-type": "beta",
  "release-type": "node",
  "versioning-strategy": "prerelease",
  "versioning-type": "prerelease",
  "versioning": "prerelease",
  "prerelease": true,
  "include-v-in-tag": true,
  "last-release-sha": "767ce15708e433867bdff190abf422d9d7defbeb",
  "initial-version": "0.61.0",
  "include-component-in-tag": true,
}
```
2. create a staging workflow on each PR merged on main. -> Release please create a PR with new features and fixes
3. If I merge this PR, it: a) create a staging tag like v1.1.0-beta b) run the release-please-config.production workflow with the 
following config:
```
{
  "release-type": "node",
  "include-v-in-tag": true,
  "last-release-sha": "767ce15708e433867bdff190abf422d9d7defbeb",
  "initial-version": "0.61.0",
  "include-component-in-tag": true,
  "pull-request-title-pattern": "chore: release Production ${version}",
}
```

Release-please create a new PR for me, with a new version v1.1.0. If I merge this PR, it creates a tag

But if I don"t merge the production PR, and I add some new bug fixes, a new staging PR is created v1.1.0-beta.1

Is it possible to create such a workflow?

I tried with 2 release-please-manifests, but I am not convinced of the way to do it

**Comment by ouraluukas** [comment]
@Spoutnik97 did you figure this out yet 👀 I want to do something similar and haven't found much info on how to do it.

**Comment by Spoutnik97** [comment]
No! So I created a new package to do what I wanted to accomplish.

You can find it as a cli https://github.com/Spoutnik97/releaser or a npm package https://www.npmjs.com/package/releaser-cli

The GitHub's action is provided.
Ask me if you need help

**Comment by airtonix** [comment]
The reason you won't find much support for this approach is that deciding which environment to deploy to has long since been agreed upon by many decades of learned trauma to be a thing to be avoided in git.

Instead all you should be focusing on in git is versions.

- merge to master === minor bump
  - merge a release branch created from master === normal release
  - also creates a release/X.Y branch so you can do the next thing.
- merge to release/X.Y === patch bump
  - merging a release/X.Y release branch === hotfix release

The outputs of all this should just be artifacts that require no further compiling etc.
There's various aways to do that : Docker, wasm, compile etc.

How you decide what version is deployed to what environment can be separatly run workflows that are not triggered by branch or tag changes and only require a version input by a human, or 
you can use something like Octopus Deploy, or you can use a separate orchestrating repo if you have other services that need deploying as a group.
